lib/valid_email2/address.rb

```diff
@@ -133,7 +140,9 @@ def mx_server_is_in?(domain_list)
     def address_contain_emoticons?
       return false if @raw_address.nil?
 
-      @raw_address.scan(Unicode::Emoji::REGEX).length >= 1
+      @raw_address.each_char.any? do |char|
```

Comment:
maybe i missed the conversation here, but why is `scan` not used any longer? i think this would be good as a comment here. also, if `permitted_multibyte_characters_regex` we should probably not do any of this, right?

Reply:
> maybe i missed the conversation here, but why is scan not used any longer?

This was a difference that came in this PR, so I simply put it back in.
https://github.com/micke/valid_email2/pull/257

I don't see any particular problem since `each_char` also handles one character at a time.

```ruby
raw_address = 'あいうえお@gmail.com'
raw_address.each_char do |char|
  print char, ' '
end

#=> あ い う え お @ g m a i l . c o m
```

Reply:
Understood.
Thank you. I will correct it. 👍

Reply:
I tried to change it. 94299bf279adf1f4d7263dfeafdd30724662287d

Since some multibyte characters are not included in the original Unicode::Emoji::REGEX ( e.g. `ø` ), 
I checked with the `bytesize` method. Please comment if you find anything strange.

Reply:
cool, thanks!